packages/wrangler/src/api/pages/deploy.tsx

```diff
@@ -248,6 +248,15 @@ export async function deploy({
 		);
 	}
 
+	// if (config.configPath !== "") {
+	// 	const configHash = createHash("sha256")
+	// 		.update(await readFile(configPath))
+	// 		.digest("hex");
+
+	// 	formData.append("wrangler-config-hash", configHash);
+	// 	formData.append("pages_build_output_dir", config.pages_build_output_dir);
+	// }
+
```

Comment:
Is there a reason this is commented out?

Reply:
yes, it's mainly because the config object doesn't exist in this file yet, but wanted to have this change ready for when it does. Currently, the config file is read in `/pages/deploy.tsx` and am trying to figure out what's the best place to actually read this file. If we keep it in `/pages/deploy.tsx` we might need to move some code over from `/api/pages/deploy.tsx` to `/pages/deploy.tsx`, specifically:

```
const project = await fetchResult<Project>(
	`/accounts/${accountId}/pages/projects/${projectName}`
);
let isProduction = true;
if (branch) {
	isProduction = project.production_branch === branch;
}

const deploymentConfig = project.deployment_configs[isProduction ? "production" : "preview"];
```

But I'm not sure I really like that, so :processing: for the time being :)